idea/idea-frontend-independent/src/org/jetbrains/kotlin/idea/quickfix/ExclExclCallFixes.kt

```diff
@@ -25,28 +26,18 @@ abstract class ExclExclCallFix(psiElement: PsiElement) : KotlinPsiOnlyQuickFixAc
 class RemoveExclExclCallFix(psiElement: PsiElement) : ExclExclCallFix(psiElement), CleanupFix, HighPriorityAction {
     override fun getText(): String = KotlinBundle.message("fix.remove.non.null.assertion")
 
-    override fun isAvailable(project: Project, editor: Editor?, file: KtFile): Boolean =
-        getExclExclPostfixExpression() != null
-
     override fun invoke(project: Project, editor: Editor?, file: KtFile) {
         if (!FileModificationService.getInstance().prepareFileForWrite(file)) return
 
-        val postfixExpression = getExclExclPostfixExpression() ?: return
+        val postfixExpression = element as? KtPostfixExpression ?: return
         val expression = KtPsiFactory(project).createExpression(postfixExpression.baseExpression!!.text)
         postfixExpression.replace(expression)
     }
 
-    private fun getExclExclPostfixExpression(): KtPostfixExpression? {
-        val operationParent = element?.parent
-        if (operationParent is KtPostfixExpression && operationParent.baseExpression != null) {
-            return operationParent
-        }
-        return null
-    }
-
     companion object : QuickFixesPsiBasedFactory<PsiElement>(PsiElement::class, PsiElementSuitabilityCheckers.ALWAYS_SUITABLE) {
         override fun doCreateQuickFix(psiElement: PsiElement): List<IntentionAction> {
-            return listOfNotNull(RemoveExclExclCallFix(psiElement))
+            val postfixExpression = psiElement.getNonStrictParentOfType<KtPostfixExpression>() ?: return emptyList()
```

Comment:
This change is needed because FE1.0 reports `UNNECESSARY_NON_NULL_ASSERTION` on the `!!` operator but FIR reports on the expression (`KtPostfixExpression`).